stdlib/2/subprocess.pyi

```diff
@@ -57,10 +57,13 @@ STDOUT = ...  # type: int
 
 class CalledProcessError(Exception):
     returncode = 0
-    cmd = ...  # type: str
-    output = ...  # type: str  # May be None
+    cmd = ...  # type: Any
+    output = ...  # type: Any
 
-    def __init__(self, returncode: int, cmd: str, output: Optional[str] = ...) -> None: ...
+    def __init__(self,
+                 returncode: int,
+                 cmd: Union[bytes, Text, List[bytes], List[Text]],
```

Comment:
Instead of List, can't it be any Sequence? That's what check_output is annotated as.